app/views/presentations/_sign_up_link.html.erb

```diff
@@ -0,0 +1,5 @@
+<%=
+  if event&.upcoming?
+    link_to 'Sign up', new_presentation_path(prepopulated_event: event.id)
```

Comment:
Cool, If it never makes sense to build a presentation in isolation form an event. Should presentations then be a nested resource of events?